app/scripts/templates/sign_in.mustache

```diff
@@ -45,8 +45,8 @@
            </form>
 
            <div class="links">
-             <a href="/reset_password" class="left reset-password" data-flow-event="forgot-password">{{#t}}Forgot password?{{/t}}</a>
-             <a href="/#" class="right use-different" data-flow-event="use-different-account">{{#t}}Use a different account{{/t}}</a><br/>
+             <a href="/reset_password{{queryString}}" class="left reset-password" data-flow-event="forgot-password">{{#t}}Forgot password?{{/t}}</a>
+             <a href="/{{queryString}}" class="right use-different" data-flow-event="use-different-account">{{#t}}Use a different account{{/t}}</a><br/>
```

Comment:
use-different re-renders the signin page, so we should change the href to be /signin with the query string. We already have equivalent functionality to `useDifferent=1`, it's `email=blank` - see https://github.com/mozilla/fxa-content-server/blob/9cbf5d91dea1e871251ffccdbdda1db6d06cb2c8/app/scripts/models/reliers/relier.js#L38 and https://github.com/mozilla/fxa-content-server/blob/9cbf5d91dea1e871251ffccdbdda1db6d06cb2c8/app/scripts/lib/constants.js#L85, no need for a new query parameter.